views/layout.hbs

```diff
@@ -99,6 +101,38 @@
 </div>
 
 
+<!-- Modal -->
```

Comment:
yeah I thought about putting on index but I think I want on all pages in case you get linked to a specific quiz from our social features and you're like "how the HECK does this work?"